core/iwasm/libraries/wasi-nn/test/build.sh

```diff
@@ -7,7 +7,7 @@
     -Wl,--allow-undefined \
     -Wl,--strip-all,--no-entry \
     --sysroot=/opt/wasi-sdk/share/wasi-sysroot \
-    -I/home/wamr/core/iwasm/libraries/wasi-nn \
+    -I/wasi-nn \
```

Comment:
Had better not include an absolute path, how about `-I..` or `-I../../wasi-nn`?

Reply:
done